lib/improver/nbhood/nbhood.py

```diff
@@ -391,3 +380,111 @@ def process(self, cube):
         merged_cube = check_cube_coordinates(
             cube, merged_cube, exception_coordinates=exception_coordinates)
         return merged_cube
+
+
+class GeneratePercentilesFromANeighbourhood(BaseNeighbourhoodProcessing):
+
+    """Class for generating percentiles from a neighbourhood."""
+
+    def __init__(
+            self, neighbourhood_method, radii, lead_times=None,
+            ens_factor=1.0, percentiles=DEFAULT_PERCENTILES):
+        """
+        Create a neighbourhood processing plugin that generates percentiles
```

Comment:
I don't think we mean create a plugin. Create an instance of the plugin maybe?

Reply:
Changed.